portfolio/src/main/webapp/script.js

```diff
@@ -53,3 +53,28 @@ function fetchBlobstoreUrlAndShowForm() {
         messageForm.classList.remove('hidden');
       });
 }
+
+/** Loads the API key from json file and attaches to html. */
+fetch('./config.json')
+    .then(response => {
+      return response.json();
+    })
+    .then(data => {
+      const script = document.createElement('script');
+      script.src =
+          'https://maps.googleapis.com/maps/api/js?key=' + data.api_key +
+          '&callback=initMap&libraries=&v=weekly';
+      script.defer = true;
+      window.initMap = function() {
+        createBasicMap();
+      };
+      document.head.appendChild(script);
+    });
+
+/** Creates the basic map. */
+function createBasicMap() {
+  map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
As Robert mentioned in my PR, maybe have a constant or some way explaining the location.

Reply:
Yes, defining the map constant makes sense! I have just updated that. I think that is what you mean, right?

Reply:
You could use the variable name to describe the location [like Annie did](https://github.com/aannie-y/step/pull/14/files).

Reply:
Sure. I have just added the location in the map name (as description) as well.